src/send_kcidb.py

```diff
@@ -24,19 +24,21 @@
 from logger import Logger
 
 
-class cmd_run(Command):
-    help = "Listen for events and send them to KCDIB"
-    args = [Args.db_config]
+class KCIDB:
 
-    def __init__(self, sub_parser, name):
-        super().__init__(sub_parser, name)
+    def __init__(self, configs, args):
         self._logger = Logger("config/logger.conf", "send_kcidb")
-
-    def __call__(self, configs, args):
         db_config = configs['db_configs'][args.db_config]
         api_token = os.getenv('API_TOKEN')
-        db = kernelci.db.get_db(db_config, api_token)
+        self._db = kernelci.db.get_db(db_config, api_token)
```

Comment:
Further `db` object calls in the script need to be changed to `self._db` i.e. lines: 66, 77, 78, 119 (64, 73, 74, 114)

Reply:
Ah yes, I think I'll just drop the refactoring of `send_kcidb.py` from this PR as I can't really test it.